### DISCUSSION TITLE
blog/vscode-error-could-not-register-service-workers-invalid-state/

### DISCUSSION THREAD

**Discussion by giscus** [jeremy-feng/giscus#5] [Announcements]
# blog/vscode-error-could-not-register-service-workers-invalid-state/

https://fengchao.pro/blog/vscode-error-could-not-register-service-workers-invalid-state/

**Comment by Elenairene** [comment]
非常有效！！

**Reply by jeremy-feng** [reply]
哈哈很高兴有帮助！

**Comment by peter-kinger** [comment]
补充：cursro 遇到问题也可以解决，感谢博主！

**Reply by peter-kinger** [reply]
cursor 打错了

**Reply by jeremy-feng** [reply]
嗯嗯，Cursor 也是基于 VS Code 开发的，解决方案自然通用。谢谢你的反馈~

**Comment by GZDDMZ** [comment]
非常有效！
Trae的相同问题也能用这个方法解决。
